frontend/src/components/nodes/ValidatorsList.tsx

```diff
@@ -0,0 +1,74 @@
+import BN from "bn.js";
+import React from "react";
+
+import * as N from "../../libraries/explorer-wamp/nodes";
+
+import ValidatorRow from "./ValidatorRow";
+
+interface Props {
+  validators: any;
+  pages: any;
+  cellCount: number;
+  validatorType: string;
+}
+
+class ValidatorsList extends React.PureComponent<Props> {
+  calculateStake = (nodeIndex: number, totalStake: BN) => {
+    let total = new BN(0);
+    let networkHolderIndex = [];
```

Comment:
You don't need an array here, you only care about the very first value. Just initialize it as null and only set it if the accumulated stake is over 33% and also this variable is still null.